<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ESA</class>
 <widget class="QWidget" name="ESA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ESA - Virtual Telecom Lab</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>16</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="startBut">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>72</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopBut">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>72</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="singleBut">
           <property name="maximumSize">
            <size>
             <width>72</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Single</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Horizontal Control</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="horizontalSpacing">
           <number>12</number>
          </property>
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Stop (MHz)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="stopSpin">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="maximum">
               <double>20000.000000000000000</double>
              </property>
              <property name="value">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Start (MHz)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="startSpin">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="maximum">
               <double>20000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Span (MHz)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="spanSpin">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>0.001000000000000</double>
              </property>
              <property name="maximum">
               <double>20000.000000000000000</double>
              </property>
              <property name="value">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Center (MHz)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="centerSpin">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="maximum">
               <double>20000.000000000000000</double>
              </property>
              <property name="value">
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Vertical Control</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QRadioButton" name="dbmRadio">
            <property name="text">
             <string>dBm</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="linRadio">
            <property name="text">
             <string>Linear (Auto)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>dB/Div</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dbdivSpin">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>0.100000000000000</double>
              </property>
              <property name="maximum">
               <double>30.000000000000000</double>
              </property>
              <property name="value">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Ref level (dBm)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="reflevelSpin">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <double>-300.000000000000000</double>
              </property>
              <property name="maximum">
               <double>100.000000000000000</double>
              </property>
              <property name="value">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Acquisition Control</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="horizontalSpacing">
           <number>12</number>
          </property>
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="3" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Spectrogram N</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sgNSpin">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1024</number>
              </property>
              <property name="value">
               <number>32</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QCheckBox" name="windowCheck">
              <property name="text">
               <string>Kaiser Window</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="windowSpin">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Points</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="pointsSpin">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>1000000</number>
              </property>
              <property name="value">
               <number>1000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Sample Rate Turbo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="srturboSpin">
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <double>1.000000000000000</double>
              </property>
              <property name="maximum">
               <double>100.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Averages</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="avgSpin">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1024</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>RBW (MHz)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="rbwSpin">
              <property name="maximumSize">
               <size>
                <width>128</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="keyboardTracking">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>4</number>
              </property>
              <property name="minimum">
               <double>0.000100000000000</double>
              </property>
              <property name="maximum">
               <double>2000.000000000000000</double>
              </property>
              <property name="value">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="peakCheck">
            <property name="maximumSize">
             <size>
              <width>128</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Peak detection</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QCheckBox" name="syncCheck">
            <property name="text">
             <string>Sync start</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_10">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="saveBut">
         <property name="maximumSize">
          <size>
           <width>2560</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <widget class="QWidget" name="normal">
        <attribute name="title">
         <string>Normal</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QGridLayout" name="graphHolder">
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="spectrogram">
        <attribute name="title">
         <string>Spectrogram</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <layout class="QGridLayout" name="sggraphHolder">
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
